app/agents/voice/automatic/tools/juspay/analytics.py

```
@@ -500,6 +500,85 @@ def get_average_ticket_payment_wise_by_time(
+                "field": "ticket_size",
+                "val": input_amount,
+            },
+        }
+
+        analytics_result = await _make_genius_api_request(params, analytics_payload)
+        if isinstance(analytics_result, ApiFailure):
+            await params.result_callback(analytics_result.error)
+            return
+
+        await params.result_callback({"data": json.dumps(analytics_result.data)})
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)